apps/meteor/app/lib/server/methods/getRoomJoinCode.ts

```diff
@@ -26,8 +27,9 @@ Meteor.methods<ServerMethods>({
 			throw new Meteor.Error('error-not-authorized', 'Not authorized', { method: 'getJoinCode' });
 		}
 
-		const [room] = Rooms.findById(rid).fetch();
+		const room = await Rooms.findById(rid);
```

Comment:
Missing toArray

Reply:
nop, i have changed the function find by id only returns one , its a find one, no need to put to array

Reply:
look
<img width="493" alt="image" src="https://user-images.githubusercontent.com/5263975/227745137-d8577583-73d3-4243-805b-5560cba147d0.png">